multi_physics/QED/include/picsar_qed/qed_commons.h

```diff
@@ -131,7 +131,9 @@
 */
 //
 #ifdef __SYCL_DEVICE_ONLY__
-    #define PXRMP_DPCPP_FIX
+#   ifndef PXRMP_DPCPP_FIX
+#       define PXRMP_DPCPP_FIX 1
```

Comment:
the fix here is the `ifndef` - I just set the `1` for consistency with what we set in WarpX for control